tutorials/continuous_aggs_tutorial.md

```diff
@@ -0,0 +1,171 @@
+
+# Tutorial: Continuous Aggregates
+
+
+A continuous aggregate query computes the query results automatically at a user specified time interval without the user having to repeatedly rerun the query.  This is particularly useful when recomputing aggregates frequently for large data sets.
```

Comment:
I think this section needs a bit more explanation of what continuous aggs are

Reply:
@gayyappan what do you think of (leaving out table and column names and things)

---

(at this point you should have read {earlier tutorials})

On kind of query that occurs frequently when performing data analysis is
aggregating data into summaries more useful for understanding what's going on.
For instance if you want to find out the avg {whatever} for each day in your
dataset you would run
```SQL
SELECT time_bucket('1 day', time), avg(data), stddev(data) FROM table;
```
However, performing this query as written requires scanning all the data within
the table, which can be inefficient if this query is called frequently.
Continuous Aggregates are a form of view which computes the query results
automatically, in the background, ...

Reply:
added